packages/web-frontend/cypress/e2e/dashboardReports.spec.js

```diff
@@ -36,7 +37,8 @@ describe('Dashboard reports', () => {
 
       cy.visit(url);
       cy.wait('@report');
-      cy.findByTestId('enlarged-dialog').snapshotHtml({ name: 'html' });
+      cy.findByTestId('enlarged-dialog').snapshotHtml({ name: SNAPSHOTS.key });
+      cy.findByTestId('enlarged-dialog').snapshotHtml({ name: SNAPSHOTS.newKey });
```

Comment:
I don't really understand this, but maybe by the time I'm finished the review I will

Reply:
We use snapshots under the "standard" key `SNAPSHOTS.key` for comparison with existing snapshots, since they will be stored in the snapshot repo (and pulled) using the same key. Snapshot comparison is done on a nested key basis

At the same time, we want to store the new snapshots somewhere avoiding a comparison... That's why we use `SNAPSHOTS.newKey`, which serves as a "temp" variable. Eventually we will extract those snapshots after the tests run, to create the snapshot PR.

The cleanest and simplest way would be to store the new snapshots in a `newSnapshots.js` file, but unfortunately the library we use [@cypress-snapshot](https://www.npmjs.com/package/@cypress/snapshot) does not allow that.

Reply:
I just realised I should capture the snapshots in reverse order!! We want to capture "new" snapshots first, since the "existing" snapshots may throw a error because of mismatch. I had done so in a past local commit... 🤔 

Also, I guess I should add a comment to make what's happening clearer